concordium-consensus/src/Concordium/GRPC2.hs

```diff
@@ -46,8 +48,11 @@ import Concordium.Crypto.SHA256 (DigestSize, Hash (Hash))
 import Concordium.Crypto.SignatureScheme (VerifyKey (..))
 import Concordium.Types.Accounts.Releases
 import Concordium.Types.Execution
+import qualified Concordium.Wasm as Wasm
 import Data.Text (Text)
 import Data.Time (UTCTime)
+import Concordium.Wasm (InstanceInfo(iiSourceModule))
+import Concordium.Types.Block (AbsoluteBlockHeight(..))
```

Comment:
I'd prefer if you did not both include Wasm qualified, and also imported some things unqualified.